packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -184,6 +185,17 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
       versionMetadata = extractVersionMetadata(providedClientVersion);
     }
 
+    if (!this.hasLoggedBrowserSerial) {
```

Comment:
question: why do we need the boolean? isn't this  already only happening once when this class gets initialized?

Reply:
Every time a client event is sent this method is executed

Reply:
Hmm. okay i suppose. I think there was another way of doing it - make it a function in the repo, and call it from the web client on INIT